.github/workflows/build.yml

```diff
@@ -15,10 +15,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
```

Comment:
Why the downgrade? Looks like support for v11 will end in Sep 2023. Would it be possible to update to LTS v17? Does this depend on other JDK versions supported in the coveo ecosystem? 
https://endoflife.date/java

Reply:
The SDK was always compiled with the version 11. I just made sure the build workflow was in sync with that version.
With that said, I think we could create a PR to bump to LTS.

It seems like [SAP supports versions 8, 11 and LTS (17)](https://sap.github.io/cloud-sdk/docs/java/frequently-asked-questions#which-java-versions-are-supported-by-the-sap-cloud-sdk).
So it would be nice to move to 17.

Reply:
The reason why we were "behind" was because, Java being often used in enterprise field, well it suffers from the enterprise bog: it's fricking slow for folks to update.

However, I didn't knew that Oracle moved to an LTS release cycle.

I note however that their LTS is very very very long: Java 8 security support will end at the very end of the decade, literally.


While support for SAP is important, it's important to keep a broader picture in mind and note that there might be another system that do not run yet on Java 17.

So, my two cents:
 - We always support the latest Active LTS
 - We attempt to support the second latest Active LTS, we reserve the right to stop support for this version, even outside of our usual major release cycle.
 - We do not guarantee nor will try to support other Java versions.